packages/theme-chalk/src/color-picker-panel.scss

```
@@ -89,6 +89,62 @@
+    padding: 0 8px;
+  }
+
+  @include e(gradient-track) {
+    position: absolute;
+    left: 4px;
+    right: 4px;
+    top: 50%;
+    transform: translateY(-50%);
+    height: 6px;
+    border-radius: 3px;
+    border: 1px solid getCssVar('border-color-lighter');
```

Stylelint (17.13.0)

[error] Expected "getCssVar" to be "getcssvar" (function-name-case)

(function-name-case)